usaspending_api/api_contracts/contracts/references/Glossary.md

```diff
@@ -0,0 +1,77 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Glossary
+
+These endpoints are used to power USAspending.gov's glossary components. 
+
+
+
+## List Glossary [/api/v2/references/glossary/{?limit,page}]
+
+This endpoint returns a list of glossary data.
+
++ Parameters
+    + `page`: `1` (optional, integer)
+        Page number to return
+    + `limit`: `10` (optional, integer)
+        Maximum number to return
+
+### List Glossary [GET]
+
++ Response 200 (application/json)
+    + Attributes (GlossaryListingObject)
+
+## Glossary Autocomplete [/api/v2/autocomplete/glossary/]
```

Comment:
Please move this endpoint to it's own API Blueprint file (...../contracts/autocomplete/Glossary.md)